app/[locale]/(dashboard)/layout.tsx

```diff
@@ -2,8 +2,9 @@
 import Footer from "../../../components/footer/Footer";
 import Header from "../../../components/header/Header";
 import React from "react";
+type DashboardLayoutProps = { children: JSX.Element };
```

Comment:
same, here,  change from type to interface

Reply:
ok